src/main/java/org/opentripplanner/model/SimpleTransfer.java

```diff
@@ -40,6 +42,10 @@ public double getEffectiveWalkDistance(){
     	return this.effectiveWalkDistance;
     }
 
+    public int getDistanceIndependentTime() {
+        return distanceIndependentTime;
+    }
+
```

Comment:
Could you add the following JavaDoc to the two method above?
```
    /**
     * The distance to walk adjusted for elevation and obstacles. This is used together
     * with the walking speed to find the actual walking transfer time. This plus
     * {@link #getDistanceIndependentTime()} is used to calculate the actual transfer given a 
     * walking speed.
     */
    public double getEffectiveWalkDistance(){
    	return this.effectiveWalkDistance;
    }

    /**
     * This is the transfer time spent NOT moving like time in in elevators, escalators and 
     * waiting on read light when crossing a street. This is used together with 
     * {@link #getEdges()} to calculate the actual-transfer-time.
     */
    public int getDistanceIndependentTime() {
        return distanceIndependentTime;
    }
```

Reply:
:+1: I've added the comment to both `Edge` and `SimpleTransfer`.

Reply:
Nice